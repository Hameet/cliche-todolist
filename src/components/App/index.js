import React, { Component } from 'react';
import logo from '../../logo.svg';
import styled from 'styled-components'
import '../../App.css';

import { Todo } from '../../containers'
import { Todolist } from '../../containers'

const StyledApp = styled.div`
  display: grid;
  font-family: 'Verdana', sans-serif;
  grid-template-areas: 'board';
  height: 100vh;
  margin: 0;
  padding: 0;
  width: 100vw;
`
StyledApp.defaultName = 'StyledApp'

class App extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <img src={logo} className="App-logo" alt="logo" />
          <h1 className="App-title">Welcome to React</h1>
        </header>
        <StyledApp>
          <div className = "body">
          <Todo/>
          <Todolist />
          </div>
      
        </StyledApp>
      </div>
    );
  }
}

export default App;


