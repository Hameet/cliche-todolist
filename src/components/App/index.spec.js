import React from 'react'
import { shallow } from 'enzyme'

import App from '.'

describe('components:App', () => {
  it('renders the App with its Css properties', () => {
    expect(toJson(shallow(<App />))).toMatchSnapshot()
  })
})

