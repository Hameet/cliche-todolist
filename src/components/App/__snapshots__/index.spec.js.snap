// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:App renders the App with its Css properties 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <h1
      className="App-title"
    >
      Welcome to React
    </h1>
  </header>
  <styled.div>
    <div
      className="body"
    >
      <Connect(TodoForm) />
      <Connect(Todolist) />
    </div>
  </styled.div>
</div>
`;
